enhancements/installer/aws-custom-edge-machineset-local-zones.md

```diff
@@ -0,0 +1,933 @@
+and the [Wavelength](https://aws.amazon.com/wavelength/features/) zones to extend the
+VPC network to the metropolitan area from big centers, and for Radio Access Network,
+respectively, to allow application developers to deliver low-latency applications
+at scale to the end users.
+
+As the resources in that locations were designed for specific workloads, and
+also have limited and more expensive cloud services, cluster operators should be
+able to create worker nodes on the edge networks on day zero operations.
+
+This enhancement proposes a new machine pool named `edge`, which will create
+Machine Sets for each availability zone on the AWS Local Zones to provision compute
+resources with `NoSchedule` taint.
+
+The EP is divided into three phases:
+
+- Phase 0: Research. Understand the Local Zones using the OKD/OCP, create a use case, and
+  provides the documentation of the installation of a cluster in an existing VPC.
+- Phase 1: Install a cluster in user provided network introducing to the installer a new
+  machine pool named `edge`. The MachineSet manifests will be created for each subnet
+  provided on `platform.aws.subnets` for Local Zones. The Machine will be protected with
+  taints to avoid running the normal platform workloads.
+- Phase 2: The installer should automate the network resource creation when the user
+  provides zone names with the type `local-zone` on the 
+  `compute[?name=="edge"].platform.aws.zones`.
+
+Phases 1 and 2 are covered by this enhancement.
+
+## Motivation
+
+OCP Install Support for AWS Local Zones will address Customer Segments where low latency and
+data locality requirements enforce as deal breaker/show-stoppers for the sales teams engagements.
+
+The resources on Local Zones were designed for applications delivering low latency to
+the end-users, in general, hosted in the data plane nodes. Furthermore, the price of
+cloud services is more expensive, and the services are limited. Finally, mixing the
+ingress router backends from different locations in the Local Zones can impact
+drastically, and directly, the end users' experience (ex: routing requests from users
+from NYC to node hosted in Local Zones in Miami, instead of New York node).
+
+As the applications using edge locations are very specific workloads, cluster admins
+want to have available the compute nodes on the board without mixing cluster components
+on those pools, and would like to specify which workloads will run for each location to
+optimize the cloud resources. For that reason, the nodes on the edge pool have taints to
+`NoSchedule`.
+
+The installer supports day 2 operation in Local Zones subnets following these conditions:
+- the VPC must have enough free CIDR blocks to create subnets in Local Zones
+- the Local Zones group must be activated manually
+- the subnets must be created manually
+- the subnet must be associated with a route table
```

Comment:
This routing allows both a local zone worker to connect to `api-int` load balancer and for control plane hosts to connect to the kubelet on local zone workers?

Reply:
Yes, it allows. It is all in the same broadcast domain, with same Security Group as regular workers.